sdk/identity/test-resources-pre.ps1

```diff
@@ -22,9 +22,9 @@ param (
     [ValidatePattern('^[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}$')]
     [string] $SubscriptionId,
 
-    [Parameter(ParameterSetName = 'Provisioner', Mandatory = $true)]
+    [Parameter(ParameterSetName = 'Provisioner')]
     [ValidateNotNullOrEmpty()]
-    [string] $TenantId,
+    [string] $TenantId = '72f988bf-86f1-41af-91ab-2d7cd011db47',
```

Comment:
TODO: fix this up

Reply:
@KarishmaGhiya or @benbp I bet this is super easy - how would I plumb the tenantID from New-TestResources (whether the default or user-provided) to this bicep file? 

My goal is to not have to prompt for it when running the script locally or in CI. many thanks!

Reply:
@maorleger I think you can delete all parameters here except `$CI` and `$RemainingArguments` as they are unused anyway.